openlibrary/templates/books/daisy.html

```diff
@@ -9,59 +10,57 @@
 <div id="contentBody">
 
     <div class="nav">
-        <a href="javascript:;" onclick="history.go(-1);"><strong>Back</strong></a>
+        <a href="javascript:;" onclick="history.go(-1);"><strong>$_("Back")</strong></a>
         |
-        <a href="/">Open Library Home</a>
+        <a href="/">$_("Open Library Home")</a>
     </div>
 
     $if not page.ocaid:
         <ul class="link">
-            <li>There isn't a DAISY file available for <a href="$page.key">$page.title</a></li>
+            <li>$_("There isn't a DAISY file available for ")<a href="$page.key">$page.title</a></li>
         </ul>
     $elif page.is_daisy_encrypted():
         <div class="message locked">
-            <h2 class="sansserif">This DAISY file is protected.</h2>
-            <p>It can only be opened on a specialized device with a key issued by the Library of Congress.</p>
+            <h2 class="sansserif">$_("This DAISY file is protected.")</h2>
+            <p>$_("It can only be opened on a specialized device with a key issued by the Library of Congress.")</p>
         </div>
         <ul class="link">
-            <li><a href="$daisy.replace('XXX', page.ocaid)"><strong>Download the DAISY.zip</strong></a> for <a href="$page.key">$page.title</a></li>
+            <li><a href="$daisy.replace('XXX', page.ocaid)"><strong>$:_('Download the DAISY.zip</strong></a> for ')<a href="$page.key">$page.title</a></li>
```

Comment:
```suggestion
            <li><a href="$daisy.replace('XXX', page.ocaid)">$:_('<strong>Download the DAISY.zip</strong></a> for <a href="%(url)s">$(title)s</a>', url=page.key, title=websafe(page.title))</li>
```

Note the use of `websafe` here since we're using the `$:` with user-entered data!

Reply:
@cdrini Should the `strong` tag go inside or outside of the i18n parens?

Reply:
Whoops! That should be a `%` :+1: